### PR TITLE
Add links to the all users and all grants pages. #157

### PR DESCRIPTION
**Pull request by mattbaumann1**

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -34,6 +34,8 @@ class User < ApplicationRecord
   validates :first_name,        presence: true
   validates :last_name,         presence: true
 
+  scope :alphabetical_order,    -> { order(last_name: :asc) }
```
Comment (jseraf) [inline app/models/user.rb]: for consistency with other scope names, :order_by_last_name

File: app/views/users/index.html.haml [hunk 2]
```diff
@@ -1,16 +1,15 @@
...
-        Email
-      %th
+        Name
       %th
+        Email
 
   %tbody
     - @users.each do |user|
       %tr
+        %td= link_to user.name, profile_path(user)
```
Comment (jseraf) [inline app/views/users/index.html.haml]: - Use one of the helper methods. 
- Profile path policy is limited to current_user. Add an edit and update action for user instead.

File: app/views/grants/index.html.haml [hunk 3]
```diff
@@ -1,21 +1,33 @@
...
       %tr
         %td= link_to grant.name, grant
+        %td= grant.state
+        %td= grant.publish_date
+        %td= grant.submission_open_date
+        %td= grant.submission_close_date
         %td= link_to 'Edit', edit_grant_path(grant) if policy(grant).edit?
         %td=  link_to 'Delete', grant, method: :delete, data: { confirm: 'Are you sure?' } if policy(grant).destroy? && grant_can_be_deleted?(grant)
 
 %br
```
Comment (jseraf) [inline app/views/grants/index.html.haml]: delete

File: config/routes.rb [hunk 4]
```diff
@@ -2,6 +2,7 @@
 
 Rails.application.routes.draw do
   devise_for :users
+  resources :users,               only: :index
```
Comment (jseraf) [inline config/routes.rb]: See previous about profiles. Add edit/update actions for user

File: app/models/user.rb [hunk 5]
```diff
@@ -34,6 +34,8 @@ class User < ApplicationRecord
...
+
   def name
```
Comment (jseraf) [inline app/models/user.rb]: this should be deleted. see helper methods.

File: app/views/grants/index.html.haml (hunk 3 above)
Comment (mattbaumann1) [reply app/views/grants/index.html.haml]: These were added after talking to @firaswehbe as to what should be included.

Comment (jseraf) [reply app/views/grants/index.html.haml]: The br tag

**Pull request merged**